.github/publish.yml

```diff
@@ -0,0 +1,48 @@
+name: publish
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+
+  build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: copy env
+        run: cp .env.default .env
+      - name: install
+        run: yarn
+      - name: build
+        run: yarn build
+      - name: test
+        run: |
+          yarn chain &
+          yarn test
+
+  publish:
+    runs-on: ubuntu-latest
+    if: ${{ github.ref == 'refs/heads/master' }}
+    needs: [build-and-test]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: copy env
+        run: cp .env.default .env
+      - name: install
+        run: yarn
+      - name: prepublish
+        run: yarn prepublishOnly
+      - name: release
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          GH_TOKEN: ${{ secrets.GH_TOKEN }}
+        run: npx semantic-release --branches master
```

Comment:
I think we should keep the yarn publish:ci:latest

Reply:
was referencing `index-coop-smart-contracts` https://github.com/IndexCoop/index-coop-smart-contracts/blob/add917bec120cce4cf0fd63d0136fbc72aad16fa/.github/workflows/publish.yaml#L48

Reply:
added here [9dd3b79](https://github.com/IndexCoop/index-protocol/pull/13/commits/9dd3b79021089e373cce190bbdfef2f906ff6249)